index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>

```

Comment:
- [ ] As specified in the project requirements, KIndly make sure your heading element h1 is styled through the class assigned to it.

You may consider saying:
```html
<h1 class="heading">Hello Microverse</h1>
```
```css
.heading {
color: pink
}
```